Apps/GatewayApi/test/unit/Services.Test/UserValidationServiceTests.cs

```diff
@@ -0,0 +1,170 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.GatewayApiTests.Services.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using HealthGateway.Common.Constants;
+    using HealthGateway.GatewayApi.Models;
+    using HealthGateway.GatewayApi.Services;
+    using Microsoft.Extensions.Configuration;
+    using Moq;
+    using Xunit;
+
+    /// <summary>
+    /// UserValidationService's Unit Tests.
+    /// </summary>
+    public class UserValidationServiceTests
+    {
+        private const string Hdid = "hdid-mock";
+
+        /// <summary>
+        /// IsPhoneNumberValidAsync returns false for invalid phone number.
+        /// </summary>
+        /// <param name="phoneNumber">The phone number to validate.</param>
+        /// <param name="expected">The expected value of the validation result.</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [InlineData("3345678901", true)]
+        [InlineData("2507001000", true)]
+        [InlineData("xxx3277465", false)]
+        [InlineData("abc", false)]
+        [Theory]
+        public async Task PhoneNumberIsNotValid(string phoneNumber, bool expected)
```

Comment:
```suggestion
        /// IsPhoneNumberValidAsync.
        /// </summary>
        /// <param name="phoneNumber">The phone number to validate.</param>
        /// <param name="expected">The expected value of the validation result.</param>
        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
        [InlineData("3345678901", true)]
        [InlineData("2507001000", true)]
        [InlineData("xxx3277465", false)]
        [InlineData("abc", false)]
        [Theory]
        public async Task ShouldValidatePhoneNumber(string phoneNumber, bool expected)
```

Documentation is out of date, since the test also checks that it correctly identifies valid numbers.